mlir/lib/Dialect/SparseTensor/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 add_subdirectory(IR)
 add_subdirectory(Transforms)
+add_subdirectory(KokkosPipelines)
```

Comment:
The pipeline targeting Kokkos should probably still be in the Pipelines/ directory to follow the existing organization (still give it its own .cpp file, though)